qiskit/algorithms/gradients/__init__.py

```diff
@@ -17,74 +17,94 @@
 
 .. currentmodule:: qiskit.algorithms.gradients
 
-
-Estimator Gradients
-===================
+Base Classes
+============
 
 .. autosummary::
    :toctree: ../stubs/
 
    BaseEstimatorGradient
-   DerivativeType
-   FiniteDiffEstimatorGradient
-   LinCombEstimatorGradient
-   ParamShiftEstimatorGradient
-   SPSAEstimatorGradient
-   ReverseEstimatorGradient
+   BaseQGT
+   BaseSamplerGradient
+   EstimatorGradientResult
+   SamplerGradientResult
+   QGTResult
 
-Sampler Gradients
-=================
+Finite Differences
+==================
 
 .. autosummary::
    :toctree: ../stubs/
 
-   BaseSamplerGradient
+   FiniteDiffEstimatorGradient
    FiniteDiffSamplerGradient
+
+Linear Combinations
+===================
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   LinCombEstimatorGradient
    LinCombSamplerGradient
+   LinCombQGT
+
+Parameter Shifting
+==================
```

Comment:
Similarly, "parameter shift rule" is the standard way to refer to these I think 🙂 
```suggestion
Parameter Shift Rules
=====================
```